package.json

```diff
@@ -2,10 +2,10 @@
   "name": "devkit-core",
   "author": "Game Closure",
   "description": "devkit engine",
-  "version": "v4.5.3-50",
+  "version": "v4.5.4",
   "repository": {
     "type": "git",
-    "url": "http://github.com/gameclosure/devkit-core.git"
+    "url": "http://github.com/CodeCombiner/devkit-core.git"
```

Comment:
let's not make this change here. Once the other repos are merged, this will automatically be correct